config/jobs/cert-manager/go-pkcs12/go-pkcs12-presubmits.yaml

```diff
@@ -0,0 +1,25 @@
+presubmits:
+  cert-manager/go-pkcs12:
+
+    - name: pull-cert-manager-go-pkcs12-verify
+      always_run: true
+      max_concurrency: 8
+      decorate: true
+      branches:
+        - master
+      spec:
+        containers:
+          - image: europe-west1-docker.pkg.dev/cert-manager-tests-trusted/cert-manager-infra-images/golang-dind:20250327-a3af8ba-1.24
+            args:
+              - runner
+              - make
+              - test
+            resources:
+              requests:
+                cpu: 2
+                memory: 4Gi
```

Comment:
Maybe we can reduce this request for resources?

Reply:
Dunno, maybe just keep it - if we doesn't pay for the resource consumption. I cannot imagine a simple go test setup requires 2 cores and 4Gi of memory. 😆